omnigibson/controllers/joint_controller.py

```diff
@@ -42,6 +46,8 @@ def __init__(
         kp=None,
         damping_ratio=None,
         use_impedances=False,
+        use_gravity_compensation=False,
+        use_cc_compensation=False,
```

Comment:
i think we still need CC compensation (And this value is accurate IIRC) -- so this should be True by default